server/src/main/java/org/apache/iotdb/db/metadata/schemaregion/SchemaRegionUtils.java

```diff
@@ -61,6 +62,10 @@ public static void deleteSchemaRegionFolder(String schemaRegionDirPath, Logger l
           String.format(
               "Failed to delete schema region folder %s", schemaRegionDir.getAbsolutePath()));
     }
+    final File storageGroupDir = schemaRegionDir.getParentFile();
```

Comment:
same

Reply:
Situation here is a bit different.  A MetadataException is thrown when deletion fail.  I suggest keep the way it is here.